liquigraph-core/src/main/java/org/liquigraph/core/configuration/ClearChecksumMode.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2014-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.liquigraph.core.configuration;
+
+/**
+ * Removes current checksums from database.
+ *
+ * On next update changesets that have already been deployed will have their checksums
+ * recomputed, and changesets that have not been deployed will be deployed.
+ */
+public enum ClearChecksumMode implements ExecutionMode {
```

Comment:
I'm not entirely sure this is a good fit.
I don't think `clear checksum` is a mode of the regular Liquigraph execution, but rather a new action.
Instead, `clearChecksums()` should be a new API on the main `Liquigraph` object

Reply:
Right, I'll add the Liquigraph.clearChecksums() and I'll remove this mode and revert the Runner interface (don't think it will be needed anymore)

Reply:
done in commit 9754dceb09ff62ed1b67b7522c0dd7b681e48feb